momaland/learning/continuous/cooperative_momappo.py

```diff
@@ -513,11 +516,16 @@ def save_actor(actor_state):
     np.random.seed(args.seed)
 
     start_time = time.time()
-    out = []
 
     # NN initialization and jit compiled functions
     env: ParallelEnv = Catch.parallel_env()
+    eval_env = deepcopy(env)
```

Comment:
This might induce errors, I'm not sure we have implemented proper copy methods in the envs. I'd rather use `Catch.parallel_env()` instead.